src/services/rover.service.js

```diff
@@ -0,0 +1,30 @@
+import axios from 'axios';
+import { format } from 'date-fns';
+
+const roverBasePath = process.env.ROVER_API_URL;
+
+const getRoverInfo = async (apiKey) => {
+  const infoResponse = await axios.get(roverBasePath, {
+    params: {
+      api_key: apiKey,
+    },
+  });
+
+  return infoResponse.data;
+};
+
+export const getRoverPhoto = async (apiKey) => {
+  const roverInfo = await getRoverInfo(apiKey);
+
+  const maxDate = new Date(roverInfo.rover.max_date);
+  const dateFormatted = format(maxDate, process.env.DATE_TEMPLATE);
```

Comment:
Not sure DATE_TEMPLATE should be placed in .env file - some kind of constants.js file is better for this